cli/smartcontract/contract_test.go

```diff
@@ -213,6 +213,110 @@ func ToMap(a []testcontract.MyPair) map[int]string {
 `, string(bs))
 }
 
+func TestDynamicWrapper(t *testing.T) {
+	smartcontract.ModVersion = "v0.0.0"
+
+	tmpDir := t.TempDir()
+	e := testcli.NewExecutor(t, true)
+
+	ctrPath := filepath.Join(tmpDir, "testcontract")
+	e.Run(t, "neo-go", "contract", "init", "--name", ctrPath)
+
+	cfgPath := filepath.Join(ctrPath, "neo-go.yml")
+	bindingsPath := filepath.Join(tmpDir, "bindings.yml")
+	manifestPath := filepath.Join(tmpDir, "manifest.json")
+	cmd := []string{"neo-go", "contract", "compile", "--in", ctrPath}
```

Comment:
Use some of the existing contracts, there's no way to init one more. For example, you may use `cli/smartcontract/testdata/verify.go` contract, it already has NEF and manifest compiled, manually compile bindings configuration (use `--bindings` flag during compilation) and add it to git. And then `TestDynamicWrapper` can use the set of resulting files to build bindings (without `verify.go` compilation step).